proc_IR.py

```diff
@@ -11,27 +11,35 @@
 clock_correction = 1.785
 filter_bandwidth = 5e3
 coh_sel = {'ph1':0,
-        'ph2':1}
+        'ph2':0}
 coh_err = {'ph1':1,# coherence channels to use for error
         'ph2':r_[0,2,3]}
 # }}}
 
 for searchstr,exp_type,nodename, postproc in [
-        ('200212_IR_3_30dBm', 'test_equip', 'signal', 
-            'spincore_IR_v1'),
+        #('200212_IR_3_30dBm', 'test_equip', 'signal', 
+         #   'spincore_IR_v1'),
+        ('w8_200309', 'test_equip',2,'ag_IR2H'),
         ]:
     fl.basename = searchstr
-    s = find_file(searchstr, exp_type=exp_type,
+    if postproc=='ag_IR2H':
+        s = find_file(searchstr,exp_type=exp_type,
+                expno=nodename,
+                postproc=postproc, lookup=postproc_dict,
+                dimname='indirect')
+    if postproc=='spincore_IR_v1':    
+        s = find_file(searchstr, exp_type=exp_type,
             expno=nodename,
             postproc=postproc, lookup=postproc_dict,
-            dimname='indirect')
-    s *= exp(-1j*s.fromaxis('indirect')*clock_correction)
+            clock_correction=clock_correction, dimname='indirect')
     #{{{filter data
-    s = s['t2':(-filter_bandwidth/2,filter_bandwidth/2)]
+    if postproc=='spincore_IR_v1':
+        s = s['t2':(-filter_bandwidth/2,filter_bandwidth/2)]
     #}}}
     #{{{hermitian function test and apply best shift
     fl.next('frequency domain before')
     fl.image(s)
+    fl.show();quit()
```

Comment:
no quit statements in the middle of the code!

Reply:
I'm going to stop here -- this means it's not ready to go

Reply:
REMEMBER this branch is for proc_90.py. Clearly the other scripts are outdated. but this branch was created I believe back in march. So it does not have proc_T1CPMG and as we noted proc_IR was really bad as well. I don't want to necessarily waste your time with having you go through the other outdated scripts. This is just for proc_90.py and the load data that accompanies it.

Reply:
But it doesn't work like that.  The changes shown here are the changes you are requesting.  Maybe you need to merge, rebase, or something like that.  Probably easiest to talk about this over zoom?

Reply:
When you ask for a pull request to get reviewed, you should look at the "files changed" page, to be sure everything is in order.

Reply:
So what is the quickest way to update all files and add in the proc_90.py?

Reply:
So I merged in bruker processing as this is the most up to date for the rest of the processing scripts. Hopefully this will fix the issue?

Reply:
we'll talk about this today.  I'm guessing that you probably wanted to merge in master, not bruker processing.  Typically, it's a good idea to merge in the branch that you are trying to merge into when you think a pull request is ready.  This will both bring the code up to date and flag any conflicts, etc.  I think just leave this alone and we'll discuss today.

Reply:
Is this taken care of?

Reply:
we're not touching this pull request until #27 is done